backend/src/main/java/com/zeros/backend/repositories/impl/InMemoryRestaurantAffiliateRepository.java

```diff
@@ -0,0 +1,25 @@
+package com.zeros.backend.repositories.impl;
+
+import com.zeros.backend.models.RestaurantAffiliate;
+import com.zeros.backend.repositories.RestaurantAffiliateRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.stream.StreamSupport;
+
+@Repository
+public class InMemoryRestaurantAffiliateRepository extends AbstractInMemoryRepository<RestaurantAffiliate> implements RestaurantAffiliateRepository {
+
+    public InMemoryRestaurantAffiliateRepository() {
+
+    }
+
+    public RestaurantAffiliate findByRestaurantId(Long restaurantId) {
```

Comment:
the class AbstractInMemoryRepository has already the method findOne(Long id) to seek entity by its id